### DISCUSSION TITLE
Email as optional attribute

### DISCUSSION THREAD

**Discussion by karanvishesh** [keycloak/keycloak#34192] [Q&A] [answered]
How can we make email as optional attribute from the keycloak admin console? Is it via user declarative profile, if yes, can you please suggest how can we do that. We are using jboss keycloak 15 image. Thanks!!

**Comment by danielFesenmeyer** [comment, marked as answer]
It seems like that's currently not supported: https://github.com/keycloak/keycloak/issues/21265

Btw, there have been lots of changes to the user profile since Keycloak 15, it makes sense to upgrade to an up-to-date version soon.

**Reply by karanvishesh** [reply]
Thanks @danielFesenmeyer ! Will update to the latest version.
